kind/cluster/cluster.yaml

```diff
@@ -0,0 +1,32 @@
+kind: Cluster
```

Comment:
Would it make sense to add comments what do you need each port for?

Reply:
I think no, it will be handled in the future by Kyma CLI, and also I personally don't know the purposes of all ports, it is just extracted from Kyma